### PR TITLE
Adding unit-test + Fix style

### PR DESCRIPTION
**Pull request by olivier-stasse**
Add unit test for test_feature_point6d
Add unit test for device
Fix style with clang

### CODE REVIEW

File: unitTesting/tools/test_device.cpp [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    if (i<37)
+      ossControlVector << ",";
+  }
+  ossControlVector << ")";
+  issControlVector.str(ossControlVector.str());
+  aDevice.setVelocitySize(38);
+  aDevice.setVelocityBounds(aLowerVelBound,anUpperVelBound);
+  aDevice.setVelocity(aStateVector);
+  aDevice.setState(aStateVector); // entry signal in position
+  aDevice.controlSIN.set(issControlVector);
```
Comment (jmirabel) [inline unitTesting/tools/test_device.cpp]: Using istringstream here is very weird and can give a wrong example to someone. Could you use `controlSIN.setConstant` and change the type of the control vector variable to `vector` ?

File: unitTesting/features/test_feature_point6d.cpp [hunk 2]
```diff
@@ -105,10 +105,24 @@ class TestPoint6d
     dynamicgraph::Vector manual;
     const std::vector<dynamicgraph::sot::MultiBound> & taskTaskSOUT
         =task_.taskSOUT(time_);
-        
+    
+    /// Verify the computation of the desired frame.
+    /// -gain *(s-sd) - ([w]x (sd -s)-vd)
```
[the remark this answers is not in the document]
Comment (gabrielebndn) [reply unitTesting/features/test_feature_point6d.cpp]: I think it's omega: angular velocity

[the remark this answers is not in the document]
Comment (jmirabel) [reply unitTesting/features/test_feature_point6d.cpp]: Which angular velocity ? The feature does not take the robot current velocity. Is it *wd* ?

[the remark this answers is not in the document]
Comment (olivier-stasse) [reply unitTesting/features/test_feature_point6d.cpp]: In FRAME_DESIRED mode this is the desired angular velocity. wd.

**Comment by gabrielebndn** [comment]
@olivier-stasse there are merge conflicts on `src/so/sot.cpp`. Could you solve them?

**Comment by olivier-stasse** [comment]
@gabrielebndn :
```
git remote add ghos https://github.com/olivier-stasse/sot-core.git
git fetch ghos
git checkout  remotes/ghos/devel -b test-pr
```
Then you can try to compile this specific branch.

**Comment by olivier-stasse** [comment]
@jmirabel commit  8442825 is fixing your comment.

**Pull request closed without merging**